app/controllers/api_controller/tags.rb

```diff
@@ -80,6 +80,7 @@ def delete_resource_tags(_type, id, _data = {})
       destroy_tag_and_classification(id)
       action_result(true, "tags id: #{id} deleting")
     rescue => err
+      raise if ActiveRecord::RecordNotFound === err
       action_result(false, err.to_s)
     end
```

Comment:
I don't have a strong opinion about this, but what about using two rescue clauses there? This way we could eliminate the `if`:

```ruby
rescue ActiveRecord::RecordNotFound
  raise
rescue => err
  action_result(false, err.to_s)
end
```